.github/workflows/publish.yml

```
@@ -37,5 +46,20 @@ jobs:
           python src/overrides.py 3.6
           uv build
       - name: Publish to PyPI
-        continue-on-error: true
+        if: env.GITHUB_REPOSITORY == 'promplate/partial-json-parser'
         run: uv publish
+      - name: Read version number
+        id: read-version
+        run: |
+          cd src/partial_json_parser
+          VERSION="$(python -c 'from version import __version__; print(__version__)')"
```

I prefer not using Python to read `__version__` here because [pdm](https://backend.pdm-project.org/metadata/#read-from-a-static-string-in-the-given-file-path) uses [regex](https://github.com/pdm-project/pdm-backend/blob/e57eaa9bcdb110354d244400e4fd7f174804b935/src/pdm/backend/hooks/version/__init__.py#L68-L80) to determine the dynamic version.

```suggestion
          VERSION="$(grep -m1 -oP '^__version__\s*=\s*"\K[^"]+' src/partial_json_parser/version.py)"
```